frontend/src/containers/DocumentList.js

```diff
@@ -89,7 +89,10 @@ class DocumentListContainer extends Component {
     const urlParams = new URLSearchParams(window.location.search);
     const page = urlParams.get('page');
 
-    if (this.lastPage && this.lastPage !== page) {
+    if (
+      (typeof this.lastPage === 'undefined' || this.lastPage) &&
```

Comment:
why not simply initializing lastViewPage=-1 in constructor?

Reply:
because I believe it would be more obvious for others to understand that. If you really want me to change that let me know. Thanks

Reply:
makes sense, but then why not:
```javascript
if(this.lastViewedPage === undefined || this.lastViewedPage !== page)
```

wouldn't that be much more clean/clear?

Reply:
Simplified the logic and ditched the check for undefined